### PR TITLE
Add an error from SQLite

### PR DESCRIPTION
**Pull request by dragonsahead**
Added a few lines to make sure that users don't run on the IOError or using the /tmp dir inside the container for SQLite

https://github.com/metabase/metabase/issues/8084

### CODE REVIEW

File: docs/troubleshooting-guide/docker.md [hunk 1]
```diff
@@ -139,6 +139,18 @@ Run `curl http://localhost:port-number-here/api/health`. This should return a re
...
+
+#### How to detect this:
+
+A message in the logs will clearly indicate an IOError/Permission denied from Java or errors from SQLite with the `org.sqlite.core.NativeDB._open_utf8` form
+
+#### How to fix this:
+
+Ensure that the file you are writing or reading from a file or folder that the user that runs Metabase has access to. E.g.:
+- if you are running Metabase as a `jar` file in your local machine or server, check the user that is running the java process
+- if you are running Metabase from the docker container, make sure that you are using the `/metabase.db` folder which is owned by the metabase user who is the one that runs the java process inside the container
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/docker.md:152]:
```suggestion
### Metabase can't write or read to/from a file or directory

#### How to detect this:

A message in the logs will clearly indicate an IOError/Permission denied from Java, or errors from SQLite with the `org.sqlite.core.NativeDB._open_utf8` form.

#### How to fix this:

Ensure that the user who is running Metabase has permission to read and write to the file or directory:

- If you are running Metabase as a JAR file in your local machine or server, check the user who is running the Java process.
- If you're running Metabase from the Docker container, make sure you're using the `/metabase.db` directory.
```

We may want to tell people how to check permissions.

File: docs/troubleshooting-guide/running.md [hunk 2]
```diff
@@ -26,3 +26,7 @@ If the Metabase instance starts and runs for a significant amount of time before
     java -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/path/to/a/directory -jar metabase-jar
 
 The `-XX:HeapDumpPath` flag is optional, with the current directory being the default. When an `OutOfMemoryError` occurs, it will dump an `hprof` file to the directory specified. These can be large (i.e. the size of the `-Xmx` argument) so ensure your disk has enough space. These `hprof` files can be read with many different tools, such as `jhat` included with the JDK or the [Eclipse Memory Analyzer Tool](https://www.eclipse.org/mat/).
+
+### Metabase cannot read or write from a file or folder (IOError)
+
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/running.md:31]:
```suggestion
### Metabase can't read or write from a file or directory (IOError)

```

**Review by jeff-bruemmer — CHANGES_REQUESTED** [review]
Thanks for including this, @paoliniluis! Just some small suggestions.

File: docs/troubleshooting-guide/running.md [hunk 3]
```diff
@@ -26,3 +26,7 @@ If the Metabase instance starts and runs for a significant amount of time before
     java -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/path/to/a/directory -jar metabase-jar
 
 The `-XX:HeapDumpPath` flag is optional, with the current directory being the default. When an `OutOfMemoryError` occurs, it will dump an `hprof` file to the directory specified. These can be large (i.e. the size of the `-Xmx` argument) so ensure your disk has enough space. These `hprof` files can be read with many different tools, such as `jhat` included with the JDK or the [Eclipse Memory Analyzer Tool](https://www.eclipse.org/mat/).
+
+### Metabase can't read or write from a file or directory (IOError)
+
+Check the issue [here](docker.md)
```
Comment (ariya) [inline docs/troubleshooting-guide/running.md:32]: I suggest not using "here" as the link text (see [this old advice](https://www.w3.org/QA/Tips/noClickHere)).
If anything, "here" isn't a good search-able keyword.

File: docs/troubleshooting-guide/running.md [hunk 4]
```diff
@@ -26,3 +26,7 @@ If the Metabase instance starts and runs for a significant amount of time before
     java -Xmx2g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/path/to/a/directory -jar metabase-jar
 
 The `-XX:HeapDumpPath` flag is optional, with the current directory being the default. When an `OutOfMemoryError` occurs, it will dump an `hprof` file to the directory specified. These can be large (i.e. the size of the `-Xmx` argument) so ensure your disk has enough space. These `hprof` files can be read with many different tools, such as `jhat` included with the JDK or the [Eclipse Memory Analyzer Tool](https://www.eclipse.org/mat/).
+
+### Metabase cannot read or write from a file or folder (IOError)
+
+If you find an error regarding file permissions when using Metabase, like not being able to read a SQLite database or a custom GeoJSON, we also have you covered in this section about [Metabase not being able to read from a file or folder](docker.html)
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/running.md:32]:
```suggestion
If you find an error regarding file permissions, like Metabase being unable to read a SQLite database or a custom GeoJSON, check out the section "Metabase can't read to/from a file or directory" in our [Docker troubleshooting guide](docker.md).
```

File: docs/troubleshooting-guide/docker.md [hunk 5]
```diff
@@ -139,6 +139,21 @@ Run `curl http://localhost:port-number-here/api/health`. This should return a re
...
+A message in the logs will clearly indicate an IOError/Permission denied from Java, or errors from SQLite with the `org.sqlite.core.NativeDB._open_utf8` form.
+
+#### How to fix this:
+
+Ensure that the user who is running Metabase has permission to read and write to the file or directory:
+
+- If you are running Metabase as a JAR file in your local machine or server, check the user who is running the Java process.
+- If you're running Metabase from the Docker container, make sure you're using the `/metabase.db` directory.
+
+If you are running metabase from the jar file in any *nix (Unix like) operating system, in order to see which user is running metabase you have to open a terminal and type `ps -uA | grep metabase`.
```
Comment (jeff-bruemmer) [inline docs/troubleshooting-guide/docker.md:155]:
```suggestion
If you're running Metabase from the JAR in any *nix (Unix like) operating system, in order to see which user is running Metabase, you have to open a terminal and type `ps -uA | grep metabase`.
```

**Review by jeff-bruemmer — APPROVED** [review]
Couple of suggestions.: including one mandatory suggestion that fixes a broken link. Thanks @paoliniluis!

**Pull request merged**